packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -226,13 +229,14 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                             ) : (
                                 <div
                                     className='p2p-my-ads__table-add'
-                                    onClick={() => {
-                                        setAdvert(advert);
-                                        my_ads_store.showQuickAddModal(advert);
-                                    }}
+                                    onClick={!general_store.is_barred && onClickAddPaymentMethod}
```

Comment:
The function also has the same if check. Can you check which is suitable?

Reply:
Oh my bad, removed that, thx for the notice!